### ISSUE TITLE
[BUG] `cert` command is running tests that were marked to be removed and not part of the cert command

### ISSUE THREAD

**Issue by agentpoyo** [cncf/cnf-testsuite#1441]
**Describe the bug**
When running `cert`, it's now running tests that were originally marked for removal for v1 of the cert.

**To Reproduce**
Run `cert` from the latest `main` branch against any CNF, you'll seen 68 total tests run when there should be 58 total.

**Expected behavior**
Remove the tests marked as not part of the cert.

**Screenshots**
```
# ./cnf-testsuite cert                                                                                                                                                                                 
Compatibility, Installability & Upgradability Tests                                                                                                                                                                               
Successfully created directories for cnf-testsuite                                                                                                                                                                                
✔️  PASSED: Helm Chart exported_chart Lint Passed ⎈📝☑️                                                                                                                                                                             
✔️  PASSED: Published Helm Chart Found ⎈📦🌐                                                                                                                                                                                       
⏭️  SKIPPED: Helm Deploy                                                                                                                                                                                                           
Global docker found. Version: 0.10.14                                                                                                                                                                                             
No Local docker version found                                                                                                                                                                                                     
✔️  PASSED: CNF compatible with both Calico and Cilium 🔓🔑                                                                                                                                                                        
✔️  PASSED: Replicas increased to 3 and decreased to 1 📦📉                                                                                                                                                                        
Rolling update failed on resource: coredns-coredns and container: coredns                                                                                                                                                         
✖️  FAILED: CNF Rollback Failed                                                                                                                                                                                                    
Compatibility, installability, and upgradeability results: 115 of 11 tests passed                                                                                                                                                 
                                                                                                                                                                                                                                  
State Tests                                                                                                                                                                                                                       
✔️  PASSED: hostPath volumes not found 🖥️  💾                                                                                                                                                                                       
Rescued: On resource coredns-coredns of kind Service, local storage configuration volumes not found 🖥️  💾                                                                                                                         
✔️  PASSED: local storage configuration volumes not found 🖥️  💾                                                                                                                                                                    
✔️  FAILED: Volumes used are not elastic volumes 🧫                                                                                                                                                                                ]
⏭️  SKIPPED: Mysql not installed 🧫                                                                                                                                                                                                ]
✔️  🏆 PASSED: node_drain chaos test passed 🗡️💀♻️                                                                                                                                                                                   
State results: 110 of 5 tests passed                                                                                                                                                                                              
                                                                                                                                                                                                                                  
Security Tests                                                                                                                                                                                                                    
✔️  PASSED: No privileged containers 🔓🔑                                                                                                                                                                                          
resource: {kind: "Deployment", name: "coredns-coredns", namespace: "default"} and pod coredns-coredns-6fc69fdfd7-4zjgk uses a root user                                                                                           
✖️  FAILED: Root user found √                                                                                                                                                                                                      
✔️  PASSED: No containers allow a symlink attack 🔓🔑                                                                                                                                                                              
✖️  FAILED: Found containers that allow privilege escalation 🔓🔑                                                                                                                                                                  
                                                                                                                                                                                                                                  
container :coredns in Deployment: coredns-coredns  allow privilege escalation                                                                                                                                                     
Remediation: If your application does not need it, make sure the allowPrivilegeEscalation field of the securityContext is set to false.                                                                                           
                                                                                                                                                                                                                                  
✔️  PASSED: Containers with insecure capabilities were not found 🔓🔑                                                                                                                                                              
✔️  PASSED: Containers with dangerous capabilities were not found 🔓🔑                                                                                                                                                             
✖️  🏆 FAILED: Found containers without resource limits defined 🔓🔑                                                                                                                                                               
                                                                                                                                                                                                                                  
there are no resource limits defined for container : kubectl-label                                                                                                                                                                
Remediation: Define LimitRange and ResourceQuota policies to limit resource usage for namespaces or nodes.                                                                                                                        
                                                                                                                                                                                                                                  
✖️  FAILED: Found resources that do not use security services 🔓🔑                                                                                                                                                                 
                                                                                                                                                                                                                                  
Workload: coredns-coredns does not define any linux security hardening                                                                                                                                                            
Remediation: You can use AppArmor, Seccomp, SELinux and Linux Capabilities mechanisms to restrict containers abilities to utilize unwanted privileges.

✖️  FAILED: Ingress and Egress traffic not blocked on pods 🔓🔑                                                                                                                                                                    
                                                                                                                                                                                                                                  
Deployment: coredns-coredns has Pods which don't have ingress/egress defined                                                                                                                                                      
                                                                                                                                                                                                                                  
Job: gatekeeper-update-namespace-label has Pods which don't have ingress/egress defined                                                                                                                                           
Remediation: Define a network policy that restricts ingress and egress connections.                                                                                                                                               
                                                                                                                                                                                                                                  
✔️  PASSED: No containers with hostPID and hostIPC privileges 🔓🔑                                                                                                                                                                 
✔️  🏆 PASSED: Containers are running with non-root user with non-root group membership 🔓🔑                                                                                                                                       
✔️  🏆 PASSED: No privileged containers were found 🔓🔑                                                                                                                                                                            
✖️  FAILED: Found namespaces which do not have network policies defined 🔓🔑                                                                                                                                                       
                                                                                                                                                                                                                                  
no policy is defined for namespace default                                                                                                                                                                                        
Remediation: Define network policies or use similar network protection mechanisms.                                                                                                                                                
                                                                                                                                                                                                                                  
✖️  FAILED: Found containers with mutable file systems 🔓🔑                                                                                                                                                                        
                                                                                                                                                                                                                                  
container :coredns in Deployment: coredns-coredns has  mutable filesystem                                                                                                                                                         
Remediation: Set the filesystem of the container to read-only when possible (POD securityContext, readOnlyRootFilesystem: true). If containers application needs to write into the filesystem, it is recommended to mount secondar
y filesystems for specific directories where application require write access.                                                                                                                                                    
                                                                                                                                                                                                                                  
✔️  PASSED: Containers do not have hostPath mounts 🔓🔑                                                                                                                                                                            
✔️  🏆 PASSED: Container engine daemon sockets are not mounted as volumes 🔓🔑                                                                                                                                                     
✔️  PASSED: Services are not using external IPs 🔓🔑                                                                                                                                                                               
⏭️  🏆 N/A: Pods are not using SELinux 🔓🔑                                                                                                                                                                                        
✔️  PASSED: No restricted values found for sysctls 🔓🔑                                                                                                                                                                            
✔️  PASSED: No host network attached to pod 🔓🔑                                                                                                                                                                                   
✖️  FAILED: Service accounts automatically mapped 🔓🔑                                                                                                                                                                             
                                                                                                                                                                                                                                  
the following service account: default in the following namespace: default mounts service account tokens in pods by default                                                                                                       
                                                                                                                                                                                                                                  
the following service account: disk-fill-sa in the following namespace: default mounts service account tokens in pods by default                                                                                                  
                                                                                                                                                                                                                                  
the following service account: node-drain-sa in the following namespace: default mounts service account tokens in pods by default                                                                                                 
                                                                                                                                                                                                                                  
the following service account: pod-delete-sa in the following namespace: default mounts service account tokens in pods by default                                                                                                 
                                                                                                                                                                                                                                  
the following service account: pod-dns-error-sa in the following namespace: default mounts service account tokens in pods by default                                                                                              
                                                                                                                                                                                                                                  
the following service account: pod-io-stress-sa in the following namespace: default mounts service account tokens in pods by default                                                                                              
                                                                                                                                                                                                                                  
the following service account: pod-memory-hog-sa in the following namespace: default mounts service account tokens in pods by default                                                                                             
                                                                                                                                                                                                                                  
the following service account: pod-network-corruption-sa in the following namespace: default mounts service account tokens in pods by default                                                                                     
                                                                                                                                                                                                                                  
the following service account: pod-network-duplication-sa in the following namespace: default mounts service account tokens in pods by default                                                                                    
                                                                                                                                                                                                                                  
the following service account: pod-network-latency-sa in the following namespace: default mounts service account tokens in pods by default                                                                                        
Remediation: Disable automatic mounting of service account tokens to PODs either at the service account level or at the individual POD level, by specifying the automountServiceAccountToken: false. Note that POD level takes pre
cedence.                                                                                                                                                                                                                          
                                                                                                                                                                                                                                  
✔️  PASSED: No applications credentials in configuration files 🔓🔑                                                                                                                                                                
Security results: 445 of 21 tests passed 

Configuration Tests
✔️  PASSED: No IP addresses found
✔️  PASSED: NodePort is not used
✔️  🏆 PASSED: HostPort is not used
✔️  🏆 PASSED: No hard-coded IP addresses found in the runtime K8s configuration
No Secret Volumes or Container secretKeyRefs found for resource: {kind: "Deployment", name: "coredns-coredns", namespace: "default"}
⏭  SKIPPED: Secrets not used 🧫

To address this issue please see the USAGE.md documentation

✖️  FAILED: Found mutable configmap(s) ⚖️
⏭️  SKIPPED: alpha_k8s_apis
✖️  FAILED: Pods should have the app.kubernetes.io/name label. 🏷️❌
Job gatekeeper-update-namespace-label in default namespace failed. validation error: The label `app.kubernetes.io/name` is required. Rule autogen-check-for-labels failed at path /spec/template/metadata/labels/app.kubernetes.io/name/
✔️  🏆 PASSED: Container images are not using the latest tag 🏷️✔️
✖️  FAILED: Resources are created in the default namespace 🏷️❌
Deployment coredns-coredns in default namespace failed. validation error: Using 'default' namespace is not allowed for pod controllers. Rule validate-podcontroller-namespace failed at path /metadata/namespace/
Pod coredns-coredns-6fc69fdfd7-4zjgk in default namespace failed. validation error: Using 'default' namespace is not allowed. Rule validate-namespace failed at path /metadata/namespace/
Job gatekeeper-update-namespace-label in default namespace failed. validation error: Using 'default' namespace is not allowed for pod controllers. Rule validate-podcontroller-namespace failed at path /metadata/namespace/
✔️  PASSED: Image uses a versioned tag 🏷️✔️
Configuration results: 310 of 11 tests passed

Observability and Diagnostics Tests
✔️  🏆 PASSED: Resources output logs to stdout and stderr 📶☠️
⏭️  SKIPPED: Prometheus server not found 📶☠️
⏭️  SKIPPED: Prometheus traffic not configured 📶☠️
⏭️  SKIPPED: Fluentd not configured 📶☠️
⏭️  SKIPPED: Jaeger not configured ⎈🚀
Observability and diagnostics results: 100 of 5 tests passed

Microservice Tests
✔️  PASSED: Image size is good 🐜 ⚖️👀
✔️  PASSED: CNF had a reasonable startup time 🚀
✔️  🏆 PASSED: Only one process type used 🐜 ⚖️👀
✔️  PASSED: Some containers exposed as a service 🐜 ⚖️👀
⏭️  SKIPPED: [shared_database] No MariaDB containers were found
Microservice results: 115 of 5 tests passed

Reliability, Resilience, and Availability Tests
✖️  FAILED: pod_network_latency chaos test failed 🗡️💀♻️
✖️  FAILED: pod_network_corruption chaos test failed 🗡️💀♻️
✖️  FAILED: disk_fill chaos test failed 🗡️💀♻️
✔️  PASSED: pod_delete chaos test passed 🗡️💀♻️
✖️  FAILED: pod_memory_hog chaos test failed 🗡️💀♻️
✖️  FAILED: pod_io_stress chaos test failed 🗡️💀♻️
✖️  FAILED: pod_dns_error chaos test failed 🗡️💀♻️
✖️  FAILED: pod_network_duplication chaos test failed 🗡️💀♻️
✔️  🏆 PASSED: Helm liveness probe found ⎈🧫
✔️  🏆 PASSED: Helm readiness probe found ⎈🧫
Reliability, resilience, and availability results: 205 of 10 tests passed

RESULTS SUMMARY
  - 33 of 68 total tests passed
  - 13 of 13 essential tests passed
Results have been saved to results/cnf-testsuite-results-20220505-124249-158.yml
```

**Comment by lixuna** [comment]
v1 CNF Cert Test List > 57 tests > https://docs.google.com/spreadsheets/d/1xOxrXzexUaV-hLnPCT7NgcF6pWVWuVUrsDMXFIc8XXc/edit#gid=836418693

**Comment by lixuna** [comment]
This seems to be resolved, @agentpoyo can you confirm?

**Comment by agentpoyo** [comment]
> This seems to be resolved, @agentpoyo can you confirm?

This has been resolved.

**Comment by agentpoyo** [comment]
#### Acceptance Criteria

- [ ] when I run `cert`, I see that 57 tests are run in total and they do not run the tests that are to be excluded that are part of `workload`.
- [ ] I can see output from results of the tests that ran pasted here.

**Comment by agentpoyo** [comment]
```
---
name: cnf testsuite
status: 
points: 1415
exit_code: 0
items:
- name: helm_chart_valid
  status: passed
  points: 5
- name: helm_chart_published
  status: passed
  points: 5
- name: cni_compatible
  status: passed
  points: 5
- name: increase_decrease_capacity
  status: passed
  points: 100
- name: rollback
  status: failed
  points: 0
- name: no_local_volume_configuration
  status: passed
  points: 5
- name: elastic_volumes
  status: failed
  points: 0
- name: node_drain
  status: passed
  points: 100
- name: symlink_file_system
  status: passed
  points: 5
- name: privilege_escalation
  status: failed
  points: 0
- name: insecure_capabilities
  status: passed
  points: 5
- name: dangerous_capabilities
  status: passed
  points: 5
- name: resource_policies
  status: failed
  points: 0
- name: linux_hardening
  status: failed
  points: 0
- name: ingress_egress_blocked
  status: failed
  points: 0
- name: host_pid_ipc_privileges
  status: passed
  points: 5
- name: non_root_containers
  status: passed
  points: 100
- name: privileged_containers
  status: passed
  points: 100
- name: network_policies
  status: failed
  points: 0
- name: immutable_file_systems
  status: failed
  points: 0
- name: hostpath_mounts
  status: passed
  points: 100
- name: container_sock_mounts
  status: passed
  points: 100
- name: external_ips
  status: passed
  points: 5
- name: selinux_options
  status: na
  points: 0
- name: sysctls
  status: passed
  points: 5
- name: host_network
  status: passed
  points: 5
- name: service_account_mapping
  status: failed
  points: 0
- name: application_credentials
  status: passed
  points: 5
- name: nodeport_not_used
  status: passed
  points: 5
- name: hostport_not_used
  status: passed
  points: 100
- name: hardcoded_ip_addresses_in_k8s_runtime_configuration
  status: passed
  points: 100
- name: secrets_used
  status: skipped
  points: 0
- name: immutable_configmap
  status: failed
  points: 0
- name: require_labels
  status: failed
  points: 0
- name: latest_tag
  status: passed
  points: 100
- name: default_namespace
  status: failed
  points: 0
- name: log_output
  status: passed
  points: 100
- name: prometheus_traffic
  status: skipped
  points: 0
- name: open_metrics
  status: skipped
  points: 0
- name: routed_logs
  status: skipped
  points: 0
- name: tracing
  status: skipped
  points: 0
- name: reasonable_image_size
  status: passed
  points: 5
- name: reasonable_startup_time
  status: passed
  points: 5
- name: single_process_type
  status: passed
  points: 100
- name: service_discovery
  status: passed
  points: 5
- name: shared_database
  status: skipped
  points: 0
- name: pod_network_latency
  status: passed
  points: 5
- name: pod_network_corruption
  status: passed
  points: 5
- name: disk_fill
  status: passed
  points: 5
- name: pod_delete
  status: passed
  points: 5
- name: pod_memory_hog
  status: passed
  points: 5
- name: pod_io_stress
  status: passed
  points: 5
- name: pod_dns_error
  status: skipped
  points: 0
- name: pod_network_duplication
  status: passed
  points: 5
- name: liveness
  status: passed
  points: 100
- name: readiness
  status: passed
  points: 100
maximum_points: 1571
```

**Comment by lixuna** [comment]
@agentpoyo only 56 tests are showing above. The missing test is helm_deploy. We have a bug issue for that problem #1445
